backend_new/test/integration/reserved-community-type.e2e-spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from '../../src/app.module';
+import { PrismaService } from '../../src/services/prisma.service';
+import { jurisdictionFactory } from '../../prisma/seed-helpers/jurisdiction-factory';
+import { stringify } from 'qs';
+import { ReservedCommunityTypeQueryParams } from '../../src/dtos/reserved-community-types/reserved-community-type-query-params.dto';
+import { reservedCommunityTypeFactory } from '../../prisma/seed-helpers/reserved-community-type-factory';
+import { ReservedCommunityTypeCreate } from '../../src/dtos/reserved-community-types/reserved-community-type-create.dto';
+import { ReservedCommunityType } from '../../src/dtos/reserved-community-types/reserved-community-type.dto';
+import { IdDTO } from 'src/dtos/shared/id.dto';
+
+describe('ReservedCommunityType Controller Tests', () => {
+  let app: INestApplication;
+  let prisma: PrismaService;
+  let jurisdictionAId: string;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    prisma = moduleFixture.get<PrismaService>(PrismaService);
+    await app.init();
+
+    const jurisdictionA = await prisma.jurisdictions.create({
+      data: jurisdictionFactory(12),
+    });
+    jurisdictionAId = jurisdictionA.id;
+  });
+
+  it('testing list endpoint', async () => {
```

Comment:
Would it be useful to have a test where we don't pass in the jurisdictionId? And have reservedCommunityTypes in the db for more than one jurisdiction so this test would return all?

Reply:
I'm not entirely sure, I feel like for SASS we wouldn't run into that case very often

I'm open to adding the test but we do have some unit tests that cover what the prisma call would look like

Reply:
I do think that we should have tests that cover when the params aren't as expected/don't exist.